fluent-package/Rakefile

```diff
@@ -436,7 +436,7 @@ class BuildTask
         gem_home = ENV["GEM_HOME"]
         ENV["GEM_HOME"] = gem_staging_dir
         ENV["INSTALL_GEM_FROM_LOCAL_REPO"] = "yes"
-        sh(bundle_command, "_#{BUNDLER_VERSION}_", "install")
+        sh(bundle_command, "_#{BUNDLER_VERSION}_", "install", "-j4")
```

Comment:
Good catch!

But it is not suitable to directly specify "-j4" (fixed value) in this context.
How about using `Etc.nprocessors`?

Reply:
I changed to use `Etc.nprocessors` at https://github.com/fluent/fluent-package-builder/pull/674/commits/c304360364410882f169a70d2e0b547d92d1de73
Thanks!!